docs/source/app_developers_guide/about_event_subscriptions.rst

```diff
@@ -0,0 +1,137 @@
+**************************
+About Events Subscriptions
+**************************
+
+An event subscription consists of an event type (the name of the event) and an
+optional list of filters for this event type.
+
+An event is “in a subscription” if the event’s ``event_type`` field matches the
+subscription’s ``event_type`` field, and the event matches any filters that are
+in the event subscription. If multiple filters are included in a subscription,
+only events that pass *all* filters will be received.
+
+
+Event Filters
+=============
+
+An event filter specifies which event attributes to match. A filter has the
+following contents:
+
+  * A string describing the filter (the filter name)
+
+  * The filter type (simple or regular expression) and match type (any or all)
+
+An event filter consists of a key and a "match string". Filters apply only to
+the event attributes that have the same key as the filter. The filter's match
+string is compared against the attribute's value according to the type of
+event filter.
```

Comment:
This is confusing because it says that a filter is composed of two parts twice, and the two parts are different between the two times they are described.

Filters operate on event attributes and are actually composed of three parts:
- The "key" of the attribute that the filter applies to
- The "match string" that the value of the attribute is compared against
- The "filter type" which is used to decide the rules for comparison against the match string and the attribute value.